src/transferlistwidget.cpp

```diff
@@ -741,6 +754,10 @@ void TransferListWidget::displayListMenu(const QPoint&)
     connect(&actionForce_recheck, SIGNAL(triggered()), this, SLOT(recheckSelectedTorrents()));
     QAction actionCopy_magnet_link(QIcon(":/Icons/magnet.png"), tr("Copy magnet link"), 0);
     connect(&actionCopy_magnet_link, SIGNAL(triggered()), this, SLOT(copySelectedMagnetURIs()));
+
+    QAction actionCopy_caption_list(tr("Copy name"), 0);
```

Comment:
Why did you say it only works in Linux? In my Windows build of qB, I can see there is a "Copy magnet link" option in the menu, which I would presume is using line 424:

     qApp->clipboard()->setText(magnet_uris.join("\n"));

This is the same as the line you quoted, except the logic done on `captions` instead of `magnet_uris`.

Unless the Windows build has a different bit of code for this line.